assets/exception_file.json

```diff
@@ -0,0 +1,3 @@
+{
+
```

Comment:
if you have a fixed number of files, then the exclusion mechanism is redundant. I suggest depricking him.